src/cdk/scrolling/virtual-scroll-adapter.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {Observable} from 'rxjs';
+import {ListRange} from '../collections';
+
+export interface CdkVirtualScrollAdapter<T> {
+  dataStream: Observable<T[] | ReadonlyArray<T>>;
```

Comment:
@mmalerba shouldn't we call this a `dataSource` for consistency with some of the other components?

Reply:
Actually this is different than the `dataSource`, in the case of `cdkVirtualForOf` this stream _is_ created by connecting to the data source, but it doesn't have to be.